### PR TITLE
feat(sdk-coin-inj): inj sdk init

### PR DESCRIPTION
**Pull request by DinshawKothari**
Ticket: BG-79064

### CODE REVIEW

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]
Blocked until rel/latest back merge.

**Comment by DinshawKothari** [comment]
Closed this due to build issues.

**Pull request closed without merging**
